src/components/ScmRepositoryDetailPage/ScmRepositoryDetailPage.tsx

```diff
@@ -82,9 +95,7 @@ export const ScmRepositoryDetailPage = () => {
           </ToolbarItem>
         </Toolbar>
         <ContentBox borderTop>
-          <div style={{ width: '100%', height: '30vh', textAlign: 'center', paddingTop: '30px' }}>
-            TODO: Add Usages table here and remove the placeholder
-          </div>
+          <BuildConfigsList {...{ serviceContainerBuildConfigs: serviceContainerBuildConfigs, componentId }} />
```

Comment:
`serviceContainerBuildConfigs: serviceContainerBuildConfigs` => `serviceContainerBuildConfigs`

Reply:
Do you mean `<BuildConfigsList {...{ serviceContainerBuildConfigs, componentId }} />` instead?

Reply:
yes, indeed